src/metabase/task/sync_databases.clj

```diff
@@ -206,6 +206,8 @@
       ;; See https://www.nurkiewicz.com/2012/04/quartz-scheduler-misfire-instructions.html for more info
       (cron/with-misfire-handling-instruction-do-nothing)))))
 
+;; called [[from metabase.models.database/schedule-tasks!]] from the post-insert and the pre-update
+#_ {:clj-kondo/ignore [:unused-private-var]}
 (s/defn ^:private check-and-schedule-tasks-for-db!
```

Comment:
If this is used in another namespace we should probably just make it public. Otherwise someone might think "hey it's private, I can rename it or delete it or whatever" and it's going to cause confusion when that breaks something

Reply:
```
    ;; this is done this way to avoid circular dependencies
    (classloader/require 'metabase.task.sync-databases)
    ((resolve 'metabase.task.sync-databases/check-and-schedule-tasks-for-db!) database)
```
it's used but there (or perhaps were?) circularity issues.

Reply:
Yeah. I think we should make it public because it's used elsewhere even if we aren't using it directly without dumb `resolve` hacks